Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -91,32 +101,75 @@ void HiresTexture::Update()
 
   const std::string& game_id = SConfig::GetInstance().GetGameID();
   const std::string texture_directory = GetTextureDirectory(game_id);
-  std::vector<std::string> extensions{
-      ".png", ".bmp", ".tga", ".dds",
-      ".jpg"  // Why not? Could be useful for large photo-like textures
-  };
+  std::vector<std::string> extensions{".dds", ".png", ".bmp",  ".tga",  ".tgb",
+                                      ".jpg", ".jpe", ".jpeg", ".jfif", ".hdr"};
```

Comment:
Why? I thought we were supposed to be heading in the opposite direction?

Reply:
This will not be removed, you have no support and did not present any technical reason and/or explanation why such an artificial limit is required or recommended for whatever purpose. This is required for the per-extension console log counts to work at all.